Sources/Orbit/Components/ListChoiceGroup.swift

```diff
@@ -11,28 +20,79 @@ public struct ListChoiceGroup<Content: View>: View {
 
     @Environment(\.horizontalSizeClass) var horizontalSizeClass
 
-    let status: Status?
-    let backgroundColor: Color?
+    let title: String
+    let iconContent: Icon.Content
+    let style: ListChoiceGroupStyle
+    let titleStyle: Header.TitleStyle
     let content: () -> Content
 
     public var body: some View {
-        Card(spacing: 0, padding: 0, style: .iOS, status: status, backgroundColor: backgroundColor) {
-            content()
+        switch style {
+            case .card(let status, let backgroundColor):
+                Card(title, spacing: 0, padding: 0, style: .iOS, status: status, backgroundColor: backgroundColor) {
+                    content()
+                }
+            case .borderless:
+                VStack(alignment: .leading, spacing: .medium) {
+                    header
+
+                    VStack(alignment: .leading, spacing: 0) {
+                        content()
+                    }
+                }
+                .background(Color.white)
+                .frame(maxWidth: Layout.readableMaxWidth, alignment: .leading)
+                .frame(maxWidth: .infinity)
+                .padding(.horizontal, horizontalPadding)
+        }
+    }
+
+    @ViewBuilder var header: some View {
+        if isHeaderEmpty == false {
+            Header(title, iconContent: iconContent, titleStyle: titleStyle)
```

Comment:
`Header` should be able to decide emptiness on its own now (and return EmptyView in that case) - if not, we should fix it to avoid wrapping Orbit components with ifs.